module/os/freebsd/zfs/zfs_vnops_os.c

```diff
@@ -29,7 +29,7 @@
 /* Portions Copyright 2007 Jeremy Teo */
 /* Portions Copyright 2010 Robert Milkowski */
 
-
+#include <security/mac/mac_framework.h>
```

Comment:
I don't know the motivation of the move, but style(9) says:
```
     Kernel include files (sys/*.h) come first.  If <sys/cdefs.h> is needed
     for __FBSDID(), include it first.  If either <sys/types.h> or
     <sys/param.h> is needed, include it before other include files.
     (<sys/param.h> includes <sys/types.h>; do not include both.) Next,
     include <sys/systm.h>, if needed.  The remaining kernel headers should be
     sorted alphabetically.
```

Reply:
Pushed right before vfs.h
simd.h must be included before vnode.h

Reply:
No need to `#ifdef` before `#undef`.